subsys/bluetooth/audio/ascs.c

```diff
@@ -39,6 +39,14 @@ LOG_MODULE_REGISTER(bt_ascs, CONFIG_BT_ASCS_LOG_LEVEL);
 #include "pacs_internal.h"
 #include "cap_internal.h"
 
+#define MAX_ASES_SESSIONS CONFIG_BT_MAX_CONN * \
+				(CONFIG_BT_ASCS_ASE_SNK_COUNT + \
+				 CONFIG_BT_ASCS_ASE_SRC_COUNT)
+
+BUILD_ASSERT(CONFIG_BT_ASCS_MAX_ACTIVE_ASES <= MAX(MAX_ASES_SESSIONS),
+						   CONFIG_BT_ISO_MAX_CHAN),
+	     "Too many allocated ASEs");
```

Comment:
```suggestion
	     "Active ASEs %u larger than actual number of ASEs (%u) or ISOs (%u)", CONFIG_BT_ASCS_MAX_ACTIVE_ASES, MAX_ASES_SESSIONS, CONFIG_BT_ISO_MAX_CHAN);
```

Reply:
I have made the message more informative, but we cannot use format string for BUILD_ASSERT as I understand

Reply:
Ah, right, forgot about that part :D